### ISSUE TITLE
Consider changing Download type == "Icon" to "Logo"

### ISSUE THREAD

**Issue by hansioan** [bio-tools/biotoolsSchema#201]
@joncison @hmenager @matuskalas @piotrgithub1 @redmitry 

I had a discussion with some members of EJPRD who wanted to add some resources to bio.tools and they would like to have the possibility to store the tool logo somewhere in bio.tools so they can retrieve it and display it on their own website. 

In regards to the biotoolsSchema we have a `downloadType == "Icon"` with the definition `Icon of the software.`.
I am not sure if this refers to a favicon for a website or to an actual logo.

If it the intended use was for a logo then we should change the value of `Icon` to `Logo`, it makes more sense and there are no tools that are actually annotated with `Icon` download type. We can keep both in the beginning and phase out the `Icon` one.

If the intended use was for a favicon , like you see in the tabs of webpages, then we should rename it to `Favicon` and additionally add a `Logo` download type.

@redmitry If a change like this happens to the schema are you willing to implement it in the XSLT stuff?

Thanks

**Comment by redmitry** [comment]
> If a change like this happens to the schema are you willing to implement it in the XSLT stuff?
I am not sure what you are talking about. I do to use either XSLT nor XML. From my understanding XML Schema is abandoned by the bio.tools?

**Comment by hansioan** [comment]
> > If a change like this happens to the schema are you willing to implement it in the XSLT stuff?
> 
> I am not sure what you are talking about. I do to use either XSLT nor XML. From my understanding XML Schema is abandoned by the bio.tools?

@redmitry 
It's not abandoned, it will probably be replaced by a JSON schema, but not anytime soon. 
The XSLT and XML are used for transforms and verification. If I remember correctly we had a discussion at some point about the XML schema and I asked you if you were willing to help maintain it. Perhaps I didn't understand correctly.

**Comment by matuskalas** [comment]
Indeed, it's a good idea to change the download type to "Logo", good now that nobody so far used the "Icon".

And for a favicon? I can imagine that it might be useful in the future to also be able to store a link to a small version of the logo, usable as an icon, in case of some big complicated logoi. Sounds again like a hierarchy of the types would be useful. But definitely not urgent anytime soon ;) (Future UX)

If needed, I can help with the XSD. (But not right these 2 weeks if anything bigger)
